tests/integration/test_ols.py

```diff
@@ -45,19 +47,10 @@ def test_status() -> None:
     }
 
 
+# the raw prompt should just return stuff from LLMChain, so mock that base method in ols.py
+@patch("ols.app.endpoints.ols.LLMChain", new=mock_llm_chain("test response"))
 def test_raw_prompt(monkeypatch) -> None:
```

Comment:
You can remove `monkeypatch` here. It's not used anymore.

Reply:
good catch, we'll get it cleaned up in a follow up